recipes/adam_core/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "adam_core" %}
+{% set version = "0.4.0" %}
+{% set python_min = "3.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/adam_core-{{ version }}.tar.gz
+  sha256: a1b9cf9048e4cc578b35a3ee42076af52eed35571e6d1db4af5c144445ae3cf6
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: true  # [win or py<311]
```

Comment:
This one can probably be noarch. What is the reason to skip Windows?

It is OK for noarch package to fails on Windows with a missing dependency. We prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on Windows.

Reply:
From my previous comment:
> This package skips windows as it appear some of the underlying dependencies are not available.
My understanding is I had to remove the noarch in order to do the windows skip.
Without noarch, it appears I am not allowed to set the python_min pinning in the same way.
I believe using the py<311 skip is the correct way to do this in this circumstance when a minimum python version is still required, but I welcome feedback.

I am happy to revert back to noarch and let the windows build fail due to missing dependencies. I didn't think it would get merged in with a failing build.